### PR TITLE
PT-1335 Display legal stuff in YourLoops (such as CE mark)

### PR DESCRIPTION
**Pull request by yvon-dblg**
image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -35,8 +36,11 @@ packageMounts/*
 _book/
 
 yarn.lock
-package-lock.json
 
 # Ignore old translation files
 locales/*/*_old.json
 locales/*/parameter.*.json*
+
```
Comment (BigFab) [inline .gitignore:43]: in theory, this should not be needed here, right?

File: locales/en/translation.json [hunk 2]
```diff
@@ -477,6 +477,7 @@
   "Readings In Range": "Readings In Range",
   "Ready!": "Ready!",
   "Refresh": "Refresh",
+  "Regulatory Informations": "Regulatory Informations",
```
Comment (franck-fourel) [inline locales/en/translation.json:480]:
```suggestion
  "Regulatory Informations": "Intended Use",
```

File: .gitignore (hunk 1 above)
Comment (yvon-dblg) [reply .gitignore:43]: yes. Just I tried to test them.
So they were showed as new file. To be sure to not commit them.
Anyway it doesn't hurt

**Comment by yvon-dblg** [comment]
> 
> 
> new translations items are missing in the french and german versions.
It will be done via Crowdin.
Crowdin will have them erased anyway.
> Still wondering if we make more changes for the small screens. See slack conversation.
Yes, I forgot that point. I will fix it.

File: app/core/less/containers.less [hunk 3]
```diff
@@ -40,15 +41,8 @@
...
-  padding-left: ~"calc((100% - 275px) / 2)";
-  padding-right: ~"calc((100% - 275px) / 2)";
-
-  @media(min-width: @screen-lg-min) {
-    padding-left: 0;
-    padding-right: 0;
-  }
-}
+// .container-nav-inner {
+// }
```
Comment (franck-fourel) [inline app/core/less/containers.less:45]: still used in footerlink.js

File: app/core/less/nav.less [hunk 4]
```diff
@@ -49,6 +49,4 @@
 
 .nav-right {
   float: right;
-
-  margin-right: -@nav-item-horizontal-padding;
-}
\ No newline at end of file
+}
```
Comment (franck-fourel) [inline app/core/less/nav.less]: this change is moving the signup link to the top right corner. 
Is it expected to be there ?

File: app/core/less/containers.less (hunk 3 above)
Comment (yvon-dblg) [reply app/core/less/containers.less:45]: I kept this entry comment if we need to set something in it after.
I will remove the class in the footer.

File: app/core/less/nav.less (hunk 4 above)
Comment (yvon-dblg) [reply app/core/less/nav.less]: Not from this change, acceptable for now.
We will see later if we want another layout.

**Pull request merged**